db/migrate/20170808063110_create_framework_tenders.rb

```diff
@@ -0,0 +1,16 @@
+class CreateFrameworkTenders < ActiveRecord::Migration[5.0]
+  def change
+    create_table :framework_tenders do |t|
+      t.integer :year
+      t.integer :half_year
+      t.integer :starting_month
+      t.integer :ending_month
+      t.integer :status
+      t.text :remark
+      t.integer :created_by
+      t.integer :modified_by
+      t.datetime :deleted_at
+      t.timestamps
```

Comment:
@dawitd @nathnael 
Since we're using Paranoid gem on all objects (inherited from ApplicationRecord), it will require the field deleted_at on every model otherwise we risk application error in the middle of the process.